python/dgl/heterograph_index.py

```diff
@@ -839,4 +997,24 @@ def create_heterograph(metagraph, rel_graphs):
     """
     return _CAPI_DGLHeteroCreateHeteroGraph(metagraph, rel_graphs)
 
+def create_heterograph(graph_data):
+    """Entry point of creating heterograph index from various type of
+    graph data.
+
+    Parameters
+    ----------
+    graph_data : graph data
+        Data to initialize graph structure.
+
+    Returns
+    -------
+    HeteroGraphIndex
+    """
+    if isinstance(graph_data, HeteroGraphIndex):
+        return graph_data
+    elif isinstance(graph_data, DGLGraph):
+        return create_heterograph_from_homo(graph_data)
```

Comment:
Where is this defined/imported?

Reply:
Also for DGLGraph

Reply:
I removed this. Currently, a heterograph is created via functions in `convert.py`. The class initializer is not directly exposed so the first argument right now must be HeteroGraphIndex.